torch/distributions/kl.py

```diff
@@ -278,6 +279,31 @@ def _kl_laplace_laplace(p, q):
     return t1 + t2 + t3 - 1
 
 
+@register_kl(MultivariateNormal, MultivariateNormal)
+def _kl_multivariatenormal_multivariatenormal(p, q):
+    if p.batch_shape != q.batch_shape:
```

Comment:
I would think you would error if `.event_shape` differed, and broadcast if `.batch_shape` differed.

Reply:
I forgot to add the condition about `.event_shape`. I didn't know if the intended behaviour was to broadcast. Thank you for letting me know.